### PR TITLE
Milestone #1: Calculator

### PR DESCRIPTION
**Pull request by lucasmazo32**
Hello!

I want to let you know that the link to the Heroku page is in the README file under the title "Live version".

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,5 +30,14 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "big.js": "^5.2.2",
```
Comment (Oghenebrume50) [inline package.json]: Bg.js should not be a dev dependencies

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED 

Hi @lucasmazo32 good job setting this up it looks good for a start just a few changes to be made :slightly_smiling_face:  
- Add the prop-types package as a dependency see this

Please make changes and submit for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :world_map:

**Comment by lucasmazo32** [comment]
- Made the changes in the dependencies.

**Review by MauricioRobayo — APPROVED** [review]
Great, thanks for taking care of the previous issues.

Looks good, please move on 👍 .

Status: __Approved__.

**Pull request merged**